README.md

```diff
@@ -1,28 +1,29 @@
-# The Topology Builder for Apache Kafka (Automation, Gitops, SelfService)
+# An operational manager for Apache Kafka (Automation, Gitops, SelfService)
+=======
 
 <a href="https://codeclimate.com/github/purbon/kafka-topology-builder/maintainability"><img src="https://api.codeclimate.com/v1/badges/ef4bcda7d1b5fd0a4f1e/maintainability" /></a> ![CI tests](https://github.com/kafka-ops/kafka-topology-builder/workflows/CI%20tests/badge.svg?branch=master) [![Gitter](https://badges.gitter.im/kafka-topology-builder/community.svg)](https://gitter.im/kafka-topology-builder/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge) [![Documentation Status](https://readthedocs.org/projects/kafka-topology-builder/badge/?version=latest)](https://kafka-topology-builder.readthedocs.io/?badge=latest)
 
-Topology Builder for Apache Kafka helps you automate the management of your things within Apache Kafka, from Topics, Configuration to Metadata but as well
-Access Control, Schemas. 
+Julie Ops helps you automate the management of your things within Apache Kafka, from Topics,
```

Comment:
out of curiousity, would `JulieOps` would be a better fit than `Julie Ops`, given `DevOps`, `GitOps`, etc.